src/plugins/intel_gpu/src/graph/impls/ocl/fully_connected.cpp

```diff
@@ -60,13 +60,14 @@ struct fully_connected_impl : typed_primitive_impl_ocl<fully_connected> {
                 feature = input0_pshape[primitive->input_size - 1ul];
             }
 
+            // TO DO, to remove WA
             if (primitive->input_size > 3) {
                 input0_layout.set_partial_shape(reshape_to_2d(input0_pshape, feature, primitive->input_size));
                 input0_layout.format = format::bfyx;
             }
             if (input1_pshape.size() != 2) {
                 input1_layout.set_partial_shape(reshape_to_2d(input1_pshape, feature, primitive->weights_rank));
-                input1_layout.format = format::bfyx;
+                // input1_layout.format = format::bfyx;
```

Comment:
@kelvinchoi-intel  Could you review this point?

Reply:
This doesn't matter for swin-transformer and twin-fransformer models